src/jdk.jfr/share/classes/jdk/jfr/internal/dcmd/AbstractDCmd.java

```diff
@@ -191,7 +192,7 @@ protected final void print(String s) {
     }
 
     protected final void print(String s, Object... args) {
-        currentLine.append(String.format(s, args));
+        currentLine.append(args != null && args.length > 0 ? String.format(s, args) : s);
```

Comment:
Why is args != null check needed?

(Fail fast may be preferable if something unexpected happens)

Reply:
This check is not necessary(args is not null even if no any arg passes in), removed.